### PR TITLE
Add a hyperlink to text docs.ansible.com for testing site

### PR DESCRIPTION
**Pull request by bmillemathias-1a**
##### SUMMARY
Add a hyperlink to text docs.ansible.com for testing site

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
docs.ansible.com

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by bmillemathias-1a** [comment]
+docsite

**Comment by acozine** [comment]
@bmillemathias-1a great idea! The current branch doesn't quite do the right thing when published:
Screen Shot 2020-07-23 at 11 49 42 AM

File: docs/docsite/_themes/sphinx_rtd_theme/ansible_banner.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     if (location.host == "docs.testing.ansible.com") {
       document.write('<div id="testing_banner_id" class="admonition important">');
       para = document.createElement('p');
-      banner_text=document.createTextNode("This is a testing site. For the official docs go to https://docs.ansible.com/");
+      banner_text=document.innerHTML('<b>This is the testing site for Ansible Documentation<b/>. For the official documentation website visit <a href="https://docs.ansible.com/">https://docs.ansible.com/</a>');
```
Comment (acozine) [inline docs/docsite/_themes/sphinx_rtd_theme/ansible_banner.html]: Should the end-tag be `</b>` instead of `<b/>`?

Comment (acozine) [inline docs/docsite/_themes/sphinx_rtd_theme/ansible_banner.html]:
```suggestion
      banner_text=document.innerHTML('<b>This is the testing site for Ansible Documentation</b>. Unless you're reviewing pre-production changes, please visit the <a href="https://docs.ansible.com/ansible/latest/">official documentation website</a>');
```

**Comment by bmillemathias** [comment]
image

I tuned on my local copy the css to space properly the two banners

File: docs/docsite/_themes/sphinx_rtd_theme/ansible_banner.html (hunk 1 above)
Comment (bmillemathias) [reply docs/docsite/_themes/sphinx_rtd_theme/ansible_banner.html]: indeed

File: docs/docsite/_themes/sphinx_rtd_theme/ansible_banner.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
     if (location.host == "docs.testing.ansible.com") {
       document.write('<div id="testing_banner_id" class="admonition important">');
       para = document.createElement('p');
-      banner_text=document.createTextNode("This is a testing site. For the official docs go to https://docs.ansible.com/");
+      para.innerHTML('<b>This is the testing site for Ansible Documentation</b>. For the official documentation website visit the <a href="https://docs.ansible.com/latest/">official documentation website</a>.');
```
Comment (felixfontein) [inline docs/docsite/_themes/sphinx_rtd_theme/ansible_banner.html]:
```suggestion
      para.innerHTML = '<b>This is the testing site for Ansible Documentation</b>. For the official documentation website visit the <a href="https://docs.ansible.com/latest/">official documentation website</a>.';
```

**Comment by samccann** [comment]
Thanks for the hard work here @bmillemathias-1a !  I had to add another banner update, so I absorbed this work into https://github.com/ansible/ansible/pull/71360 and will close this one.

**Comment by bmillemathias** [comment]
> 
> 
> Thanks for the hard work here @bmillemathias-1a ! I had to add another banner update, so I absorbed this work into #71360 and will close this one.

hard work, hum hum :)

**Comment by felixfontein** [comment]
Sorry for only spotting that now, but the banner links to https://docs.ansible.com/latest/ which is 404. I guess it should be https://docs.ansible.com/ansible/latest/ ?

**Comment by bmillemathias** [comment]
> 
> 
> Sorry for only spotting that now, but the banner links to https://docs.ansible.com/latest/ which is 404. I guess it should be https://docs.ansible.com/ansible/latest/ ?

I saw in #71360 @samccann put the correct URL.

**Pull request closed without merging**